_maps/map_files/stations/boxstation.dmm

```diff
@@ -40181,9 +40181,7 @@
 	d2 = 2;
 	icon_state = "0-2"
 	},
-/obj/machinery/power/apc/directional/north{
-	shock_proof = 1
-	},
+/obj/machinery/power/apc/important/directional/north,
```

Comment:
why are you removing the shock_proof on these apcs?

Reply:
Consistency - All other stations do not have shock_proof set on Engineering APCs other than the Engine APCs.
Also
![Screenshot (3)](https://github.com/user-attachments/assets/1a764bd1-3385-4d34-bad5-5e185fc38cbb)